app/models/excursion.rb

```diff
@@ -0,0 +1,13 @@
+class Excursion < ApplicationRecord
+  validates_presence_of :title
+  validates_presence_of :description
+  validates_presence_of :location
```

Comment:
You can consolidate this into a single line if you're validating the same thing (ie presence) of multiple attributes

Reply:
Good catch! We'll add this onto on current work

Reply:
Fixed and consolidated!